app/bundles/FormBundle/Views/Field/field_helper.php

```diff
@@ -161,6 +161,16 @@
         }
     }
 
+    if ($field['leadField'] && in_array($contactFields[$field['leadField']]['type'], ['datetime', 'date'])) {
+        $tempLeadFieldType = $contactFields[$field['leadField']]['type'];
+        $mauticParameter   = $this->container->getParameter('mautic.parameters');
+        foreach ($parseList as $key => $aTemp) {
+            $date = date_create_from_format('Y-m-d', $aTemp['label']);
```

Comment:
In case of datetime field type, someone might want also time there. Would it break if the date format would be `Y-m-d H:i:s`?

Also, is there a reason why it's taking label as the date source and not the key? I can imagine that someone else will want to display some different date format, but he can use `Y-m-d` format as the radio name/key.